src/components/media-views.js

```diff
@@ -734,6 +742,23 @@ AFRAME.registerComponent("media-image", {
     contentType: { type: "string" },
     batch: { default: false }
   },
+  init(){
+    this.chestImage = this.chestImage.bind(this);
+
+    this.el.setAttribute("hover-menu__image", { template: "#image-hover-menu", dirs: ["forward", "back"] });
+    this.el.components["hover-menu__image"].getHoverMenu().then( menu => {
+      if (!this.el.parentNode) return;
+
+      this.chestImageButton = this.el.querySelector(".video-chest-image-button");
+      this.chestImageButton.object3D.addEventListener("interact", this.chestImage);
+    });
+  },
+  chestImage() {
+    this.el.sceneEl
+      .querySelector("#player-rig")
+      .querySelector(".image")
+      .setAttribute("media-loader", { src: this.data.src });
+  },
```

Comment:
Same suggestions here. Rename the chestImage function, emit an event here instead of manipulating the player-rig directly.